Lib/asyncio/unix_events.py

```diff
@@ -56,9 +56,9 @@ def close(self):
                 self.remove_signal_handler(sig)
         else:
             if self._signal_handlers:
-                warinigs.warn(f"Closing the loop {self!r} "
+                warnings.warn(f"Closing the loop {self!r} "
                               f"on interpreter shutdown "
-                              f"stage, signal unsubsription is disabled",
+                              f"stage, signal unsubscription is disabled",
```

Comment:
unsubscription -> skipping signal handlers removal

Reply:
Fixed. Is now ok?